### PR TITLE
Add GeneWalk recipe

### PR DESCRIPTION
**Pull request by bgyori**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/genewalk/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    - networkx >=2.1
+    - numpy
+    - pandas
+    - pip
+    - python
+    - scipy >=1.3.0
+  run:
+    - gensim
+    - goatools
+    - indra
```
Comment (bgruening) [inline recipes/genewalk/meta.yaml]: indra is missing and should be added at first.

**Comment by bgyori** [comment]
@bgruening do you mean I should create a new recipe for INDRA, and PR that? Is it not enough for it to be available on PyPI?

**Comment by dpryan79** [comment]
@bgyori Yes, INDRA would need to be added to bioconda first.

**Comment by bgyori** [comment]
I'm giving this another shot, now with the INDRA dependency removed (and a few new plotting-related dependencies).

**Comment by dpryan79** [comment]
It's still looking for indra

**Comment by bgyori** [comment]
I realized we'll have to make another new GeneWalk release, will try again when that's done.

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@bgyori This is now passing. Shall I merge it or do you want to do any other changes first?

**Comment by bgyori** [comment]
@dpryan79, thanks, I did another version bump to get the latest version, otherwise, I think this should be good to go.

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**